### ISSUE TITLE
The issue about dataset

### ISSUE THREAD

**Issue by gymoon10** [kreshuklab/spoco#13]
Hi, I am a master course student tudying computer vision. I opened the issue as I found a probelm while trying to run the code `spoco_train.py` for spoco.

I debugged the code as CUDA memory error kept occurring in the loss calculation part. And I found that there was a problem with the part at `datasets/cvppp.py`.

When performing the  `self.train_label_transform`, the mask is not converted into a form suitable for learning due to `transforms.RandomResizedCrop(448, scale=(0.7, 1.), interpolation=0)`. The version of torchvision was 0.13.1 in my case.

For example, even if there are five instances in the mask returned by dataloader, It was found that applying `torch.unique()` did not properly count the number of instances as below. When I replaced all `transforms.RandomResizedCrop()` to `transforms.Resize`, the train works well.

![image](https://user-images.githubusercontent.com/44194558/198533947-01930402-657f-475d-a5bc-b54e2e440acd.png)

I would appreciate it if you could consider these points if you modify the code later. Also, I recommend that those who use this code take this into consideration.

Finally, hank you very much for your research and hard work.

**Comment by wolny** [comment]
Dear @gymoon10,
thanks for showing interest in our research and trying out the code yourself. Unfortunately I could not reproduce your issue. It looks like you're using some incorrect `mask` array in the 2nd cell of your notebook, because it just shows a subsequent integer numbers... The mask you've plotted in the first cell looks correct though.

A sample code snippet you can use to experiment with the CVPPP loader and investigate further (please change the path to `root_dir`):
```
import torch
import itertools
from spoco.datasets.cvppp import CVPPP2017Dataset

ds = CVPPP2017Dataset(root_dir='/home/adrian/CVPPP2017_LSC', phase='train', spoco=True, instance_ratio=0.4, seed=11)
batch5 = itertools.islice(ds, 5)
for _, _, mask in batch5:
    print(torch.unique(mask))
```

In my case (I'm using `torchvision 0.14.0` and `pytorch 1.13.0`) the output is:
```
tensor([0, 1, 2, 3, 4, 5, 6])
tensor([0, 1, 2, 3, 4])
tensor([0, 1, 2, 3, 4, 5, 6])
tensor([0, 1, 2, 3, 4, 5, 6])
tensor([0, 1, 2, 3, 4, 5, 6])
```
